coalib/bears/Bear.py

```diff
@@ -266,7 +266,9 @@ def __json__(cls):
         """
         Override JSON export of ``Bear`` object.
         """
-        _dict = get_public_members(cls)
+        # json cannot serialize properties, so drop them
```

Comment:
@Uran198 If `_dict` doesn't contain any property object then [this](https://github.com/coala-analyzer/coala/pull/2409/files#diff-7b888ece875aa379bf8cfcebcd7cdef1R282) should be removed as well. As it was there only to filter out un-serializable properties.